content/blog/automate-boot-from-san-target-configurations-using-redfish.md

```diff
@@ -0,0 +1,277 @@
+
+\
+There are several use-cases of infrastructure automation that exist, and in this blog post we will address the aspect of configuration of OS boot for a server from a Storage Area Network (SAN) based storage array.  Specifically, we will discuss the   procedure for configuring an HPE ProLiant DL Gen10, Gen10 plus and Gen11 servers to boot from a SAN target Logical Unit Number (LUN) using the Redfish® standard DMTF API. 
+
+Note:
+
+•	The steps outlined in this document were tested on HPE DL 380 Gen10 and Gen10 plus servers with Emulex based SN1610E FibreChannel (FC) cards, HPE iLO firmware 2.81 and Service Pack for ProLiant (SPP) available as of March 2023. 
+
+•	 QLogic SN1610Q cards do not support the steps outlined in this document (as of April 2023) However, future firmware releases might enable this Redfish API based configuration method.
+
+# Pre-requisite information
+
+Before we dive into the steps for performing the automation, you will need to gather a few pieces of information, such as
+
+* HPE iLO Management port ip address and credentials
+* Storage target World Wide Number (WWN) and LUN id.  A storage administrator can typically provide you with this information.
+
+Download and install the [HPE RESTful Interface Tool](https://www.hpe.com/info/resttool)  (version 4.1.0.0 or later) on the system you will initiate the automation actions from.
+Once the RESTful Interface tool (referred to as iLOrest in the remainder of the document), you can review the current boot target and their order of execution by following the steps below
+
+`ilorest login <ilo-ip> -u <ilo-user> -p password`
+
+`ilorest bootorder`
+
+Refer to the [iLOrest documentation](https://hewlettpackard.github.io/python-redfish-utility/#bootorder-command) for more information on this command.
+
+# Configuration Steps
+
+Now that you have gathered the pre-requisite information and installed the ilorest tool, let’s discuss the detailed steps for implementing the automation.
+
+## Step 1: Ensure that the FCScanPolicy value is set to ‘AllTargets’
+
+`ilorest  get FCScanPolicy --selector Bios`
+
+Expected output: FCScanPolicy=AllTargets
+
+If the value is not set to AllTargets, you can change it using the following:
+
+`ilorest  set "FCScanPolicy"="AllTargets" --selector Bios`
+
+`ilorest commit`
+
+`ilorest reboot`
+
+After rebooting the server, you can verify that the settings have taken effect using the get FCScanPolicy command or navigating to the BIOS menus as shown in Figure 1.
+
+![](/img/picture1.png "Figure 1: FCScanPolicy configuration")
+
+Step 2: Identify the right FC Card
+The server you are trying to configure might have one or more FC cards in it, and you need to identify the right card using the steps below.
+
+List out available network adapters (includes FC cards)
+
+`ilorest rawget /redfish/v1/Chassis/1/NetworkAdapters/`
+
+The previous command returns the following output: 
+
+`{
```

Comment:
Make is a code block snippet for better visibility. Please check tips #10 of the guide here: 
https://github.com/hpe-dev-incubator/hpe-dev-portal/blob/master/docs/ContributorGuide-v2.md#tips-and-trick-using-the-cms-editor